examples/cloudrun/wait-approval/service.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       containers:
       - args:
         - server
-        image: ghcr.io/pipe-cd/helloworld:v0.30.0
+        image: gcr.io/pipecd/piped:v0.27.4
```

Comment:
```suggestion
        image: gcr.io/pipecd/helloworld:v0.27.4
```

Reply:
Thanks, I get the point now 😄 Looks like there will be no version higher than `0.27.4` from now on, right? In that case, is it okay to use different versions for different examples? I know that these are just examples but multiple versions make maintaining more costly (and a bit annoying), any idea for that 🤔

Reply:
I think we still have to publish the helloword image to GCR instead of locking the examples with the old version.
So helloworld image will be duplicated on both GCR and GHCR.
I will add the workflow to publish to GCR later.

Reply:
It's sound good, LGTM 👍